core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -2056,13 +2299,13 @@ private static BsonDocument getDocumentIdFilter(final BsonValue documentId) {
   /**
    * Adds and returns a document with a new version to the given document.
    *
-   * @param document the document to attach a new version to.
+   * @param document   the document to attach a new version to.
```

Comment:
Is checkstlye not catching any of these issues?

Reply:
Checkstyle did not. The DataSynchronizer doesn't seem to actually be formatted, as far as I can tell, and my autoformatter "went to town" on the class. We should probably file a ticket (or just make a patch) to reformat it.